src/BinSkim.Driver/RoslynAnalyzer/ILImporter.Driver.cs

```diff
@@ -0,0 +1,915 @@
+                        ImportCompareOperation(opCode);
+                        break;
+                    case ILOpcode.ldftn:
+                    case ILOpcode.ldvirtftn:
+                        ImportLdFtn(ReadILToken(), opCode);
+                        break;
+                    case ILOpcode.ldarg:
+                        ImportLoadVar(ReadILUInt16(), true);
+                        break;
+                    case ILOpcode.ldarga:
+                        ImportAddressOfVar(ReadILUInt16(), true);
+                        break;
+                    case ILOpcode.starg:
+                        ImportStoreVar(ReadILUInt16(), true);
+                        break;
+                    case ILOpcode.ldloc:
+                        ImportLoadVar(ReadILUInt16(), false);
+                        break;
+                    case ILOpcode.ldloca:
+                        ImportAddressOfVar(ReadILUInt16(), false);
+                        break;
+                    case ILOpcode.stloc:
+                        ImportStoreVar(ReadILUInt16(), false);
+                        break;
+                    case ILOpcode.localloc:
+                        ImportLocalAlloc();
+                        break;
+                    case ILOpcode.endfilter:
+                        ImportEndFilter();
+                        break;
+                    case ILOpcode.unaligned:
+                        ImportUnalignedPrefix(ReadILByte());
+                        continue;
+                    case ILOpcode.volatile_:
+                        ImportVolatilePrefix();
+                        continue;
+                    case ILOpcode.tail:
+                        ImportTailPrefix();
+                        continue;
+                    case ILOpcode.initobj:
+                        ImportInitObj(ReadILToken());
+                        break;
+                    case ILOpcode.constrained:
+                        ImportConstrainedPrefix(ReadILToken());
+                        continue;
+                    case ILOpcode.cpblk:
+                        ImportCpBlk();
+                        break;
+                    case ILOpcode.initblk:
+                        ImportInitBlk();
+                        break;
+                    case ILOpcode.no:
+                        ImportNoPrefix(ReadILByte());
+                        continue;
+                    case ILOpcode.rethrow:
+                        ImportRethrow();
+                        break;
+                    case ILOpcode.sizeof_:
+                        ImportSizeOf(ReadILToken());
+                        break;
+                    case ILOpcode.refanytype:
+                        ImportRefAnyType();
+                        break;
+                    case ILOpcode.readonly_:
+                        ImportReadOnlyPrefix();
+                        continue;
+                    default:
+                        throw new BadImageFormatException("Invalid opcode");
```

Comment:
Again, deliberately not editing this file until we figure out the layering and/or code flow.